src/components/ClientUI/styledLi.jsx

```diff
@@ -2,6 +2,7 @@ import styled from 'styled-components';
 
 const StyledLi = styled.li`
```

Comment:
Actually there's `no need`  to position your `<li>` `relative`, neither to position your `<button> and <span>`  (see comments) `absolute`.  👉 This will cause weird bugs 

My suggestion is:  

```suggestion
const StyledLi = styled.li`
	display: flex;
	align-items: center;
	justify-content: space-between;
	padding: 0 20px;
	width: 350px;
	height: 50px;
	margin: 10px;
	border-radius: 15px;
	background-color: #fc814a;
`;
```